compiler/operation.cpp

```diff
@@ -90,15 +90,13 @@ void OpInfo::init_static() {
   curP++;
 
   add_binary_op(curP, tok_eq2, op_eq2);
-  add_binary_op(curP, tok_neq2, op_neq2);
   add_binary_op(curP, tok_eq3, op_eq3);
   add_binary_op(curP, tok_neq3, op_neq3);
   add_binary_op(curP, tok_spaceship, op_spaceship);
   curP++;
 
   add_binary_op(curP, tok_lt, op_lt);
   add_binary_op(curP, tok_le, op_le);
-  add_binary_op(curP, tok_neq_lg, op_neq2);
```

Comment:
It looks like was an issue, because `<>` has same priority as `==` and `!=`
https://www.php.net/manual/en/language.operators.precedence.php
![image](https://user-images.githubusercontent.com/2287155/105983888-6d562100-60aa-11eb-80a6-580d91bd45cc.png)